src/js/application.js

```diff
@@ -0,0 +1,317 @@
+        });
+    }
+    
+    showWeatherIcons(weather) {
+        [
+            {
+                iconElement: document.querySelector(selectors.TODAY_WEATHER_ICON),
+                weather: weather.getForecastForDay(),
+            },
+            {
+                iconElement: document.querySelector(selectors.TOMORROW_WEATHER_ICON),
+                weather: weather.getForecastForDay(1),
+            },
+            {
+                iconElement: document.querySelector(selectors.SECOND_DAY_WEATHER_ICON),
+                weather: weather.getForecastForDay(2),
+            },
+            {
+                iconElement: document.querySelector(selectors.THIRD_DAY_WEATHER_ICON),
+                weather: weather.getForecastForDay(3),
+            }
+        ].forEach(({ iconElement, weather }) => {
+            const iconFile = weatherIcons[weather.icon];
+
+            iconElement.style.background = `center no-repeat url("../assets/images/weather icons/${iconFile}")`;            
+            iconElement.style.backgroundSize = 'cover';
+        });        
+    }
+    
+    showLoadedWeather() { 
+        this.showTodaysWeatherDescription(this.currentWeather.getForecastForDay());
+        this.showTemperatureForecast(this.currentWeather);  
+        this.showNextDaysOfWeek(this.currentWeather);  
+        this.showWeatherIcons(this.currentWeather); 
+    }
+
+    async setLanguage(language, updateLocation = true) {
+        this.language = language;
+        const selectLanguageElement = document.querySelector(`[value=${this.language}]`);
+        selectLanguageElement.setAttribute('selected', 'selected');
+
+        this.translateContent();
+        if (updateLocation) {
+            const location = new Location();
+            await location.defineLocationByName(this.currentLocation.cityName, this.language);
+            this.currentLocation = location.getLocationData(); 
+            
+            const time = new Time(this.currentLocation.latitude, this.currentLocation.longitude, this.currentLocation.language);
+            await time.loadLocationTimezone();
+            const timeZone = time.getTimeZone();
+            await this.setLocation(this.currentLocation, timeZone);                        
+        }
+    }
+    
+    changeLanguage(e) {
+        const language = e.target.value;
+        localStorage.setItem(localStorageItems.language, language);
+        this.setLanguage(language); 
+    }
+    
+    translateContent() {
+        const stringsToBeResolved = document.querySelectorAll('[data-content]');
+
+        stringsToBeResolved.forEach(el => {
+            el.textContent = translation[el.attributes['data-content'].value][this.language];
+        });
+    }
+
+    markAsNotActive(el) {
+        el.classList.remove(classes.ACTIVE);
+        el.classList.add(classes.NOT_ACTIVE);
+    }
+    
+    markAsActive(el) {
+        el.classList.remove(classes.NOT_ACTIVE);
+        el.classList.add(classes.ACTIVE);
+    }
```

Comment:
you can leave as is, but one more possible way how to do it
![image](https://user-images.githubusercontent.com/25482018/115583873-f5774b80-a2d2-11eb-94e3-9898f50ac061.png)